### ISSUE TITLE
Can't edit title of Steps

### ISSUE THREAD

**Issue by viktorbenei** [bitrise-io/bitrise-workflow-editor#145]
From: https://discuss.bitrise.io/t/change-step-name/1852

How can I change a name of a step? I have several script steps in a workflow, all of them named "Script" which makes it difficult to find the correct one. I think in the previous version of Workflows Editor it was possible to do.

I tried clicking the name but it opens a Workflow description editor.

**Comment by viktorbenei** [comment]
Temporary workaround: you can specify the title in `bitrise.yml` mode, e.g.:

```
    - script@1.1.3:
        title: Do anything with Script step
    - deploy-to-bitrise-io@1.2.9: {}
```

**Comment by erosdome** [comment]
fixed in v1.0.7, will be available on the website soon:

https://discuss.bitrise.io/t/workflow-editor-v1-0-7-released/1879
